ocs_ci/ocs/resources/ocs.py

```diff
@@ -233,3 +233,14 @@ def get_ocs_csv():
     log.info(f"Check if OCS operator: {ocs_csv_name} is in Succeeded phase.")
     ocs_csv.wait_for_phase(phase="Succeeded", timeout=600)
     return ocs_csv
+
+
+def check_if_cluster_was_upgraded():
+    """
+    Check whether the OCS cluster went through upgrade
+
+    Returns:
+        bool: True if the OCS cluster went through upgrade, False otherwise
+
+    """
+    return True if 'replaces' in get_ocs_csv().get().get('spec') else False
```

Comment:
if the cluster got upgraded before, it will be 'replaces' and you will fail to know if it was upgraded to 4.5 right?

Reply:
that's the intention here, to return True if the cluster got upgraded. And the indication is the existence of `replaces` under the CSV `spec`.
I hope this answers the question